package.json

```diff
@@ -90,7 +90,7 @@
     "**/globule/minimatch": "^3.1.2",
     "**/hoist-non-react-statics": "^3.3.2",
     "**/isomorphic-fetch/node-fetch": "^2.6.7",
-    "**/langchain": "^0.2.11",
+    "**/langchain": "^0.2.20",
```

Comment:
If manual evaulations succeed, do we still need this resolution? It looks like it might have been originally added in order to bump the version without also bumping other packages like `@langchain/community`.

Reply:
you can try to revert it and run the code, you will know something is wrong because you will get a bunch of type errors